code/modules/admin/create_mob.dm

```diff
@@ -11,7 +11,7 @@
 	user << browse(create_panel_helper(create_mob_html), "window=create_mob;size=425x475")
 
 /proc/randomize_human(mob/living/carbon/human/H)
-	H.gender = pick(MALE, FEMALE)
+	H.gender = pick(MALE, FEMALE, PLURAL, NEUTER)
```

Comment:
If we go with having it/its pronouns for all species, I'd rather they not spawn on NPCs and it be a player only choice. (I'm fine with NPCs spawning with they/them pronouns)